src/libraries/System.Security.Cryptography/tests/DSATests.cs

```diff
@@ -171,15 +171,15 @@ protected override void Dispose(bool disposing)
             public override void ImportParameters(DSAParameters parameters) => _dsa.ImportParameters(parameters);
             public override bool VerifySignature(byte[] rgbHash, byte[] rgbSignature) => _dsa.VerifySignature(rgbHash, rgbSignature);
             protected override byte[] HashData(Stream data, HashAlgorithmName hashAlgorithm) =>
-                (byte[])_dsa.GetType().GetMethod(
+                (byte[])typeof(DSA).GetMethod(
```

Comment:
There are multiple implementations of DSA/ECDSA so this is technically getting a different method, but since it's virtual it probably doesn't matter. Just wanted to point out for double checking.

Reply:
Thanks - I was aware of that, but as you stated, the method is declared as virtual on the `DSA` class, so unless some derived class does `new` this will work (and `new` would be really weird for a protected virtual method). If this was in product I would be much more worried, but for tests I think this is perfectly OK.